CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Populate the token "requester.departments" in the hold, request delivery and transit slips, with the data provided by the backend in the ui-checkin module. Refs UICHKIN-350.
 * Populate the token "currentDateTime" in the hold, request delivery and transit slips, with the data provided by the backend in the ui-checkin module. Refs UICHKIN-348.
 * Also support `circulation` `14.0`. Refs UICHKIN-381.
+* Added requestDate token. Refs CIRC-1784.
```

Comment:
Could you please update CHANGELOG.md how i understood we should change from  `Added requestDate token. Refs CIRC-1784.` to `*  Added requestDate token. Refs UICHKIN-384.`